.github/workflows/release.yaml

```diff
@@ -161,4 +168,3 @@ jobs:
           file: Dockerfile
           tags: ${{ steps.meta.outputs.tags }}
           cache-from: type=local,src=/tmp/.buildx-cache
-
```

Comment:
It looks like we're not testing enterprise on release, even though we've made changes to the `release.yaml`. Is the intention to have some enterprise tests run before every release? If so we should add steps for that in the above sections.

Reply:
Yes, we purposely run tests against enterprise before each release, triggered on main and tag with "v*" .

Reply:
Ah my bad, I got a bit turned around while reading the diff. No problem here actually :+1: